docs/tutorials/helm.md

```diff
@@ -0,0 +1,226 @@
+Elasticsearch cluster using the official elasticsearch Helm Chart:
+
+```sh
+# Add the offical Elasticsearch Helm repository
+helm repo add elastic https://helm.elastic.co
+
+# Update the list of available charts
+helm repo update
+
+# Deploy a "simple" Elasticsearch cluster with 3 nodes
+helm install \
+  es-quickstart \
+  elastic/elasticsearch \
+  -n development-ralph \
+  --set antiAffinity=soft
+```
+
+> By default, this Helm chart will deploy three (3) Elasticsearch nodes in
+> three (3) differents Kubernetes nodes. Since we only have a single-node
+> Kubernetes cluster, we set the `antiAffinity` value to `soft` to allow our
+> deployment scenario. Feel free to adapt this option value depending on your
+> cluster constraints.
+
+Once installed, you can check pods status using the following command (and wait
+for them to get ready):
+
+```
+kubectl get pods -l app=elasticsearch-master -w
+```
+
+Once the Elasticsearch cluster is up and running, we will need to get
+automatically generated credentials to connect other applications to the
+service.
+
+For this tutorial, we will fetch `elastic` user's password using the following
+command:
+
+```
+kubectl get secrets \
+  elasticsearch-master-credentials \
+  -ojsonpath='{.data.password}' | \
+    base64 -d
+```
+
+> In a production environment, we recommend to create a user with appropriate
+> roles and permissions depending on your application needs and use its
+> credentials.
+
+To secure Elasticsearch cluster nodes communication, during its deployment, the
+Helm chart created a self-generated root certificate used to sign Elasticsearch
+cluster node SSL certificates. We will need to provide this certificate to our
+applications that connect to the cluster. It can be fetched using:
+
+```
+kubectl get secrets elasticsearch-master-certs -ojson | \
+  jq -r ".data.\"ca.crt\"" | \
+  base64 -d
+```
+
+The final step of the Elasticsearch cluster preparation is to create an index
+to store our xAPI statements. As the Elasticsearch service is not exposed, we
+will create a bridge from our machine to the Elasticsearch cluster using the
+`kubectl port-forward` command:
+
+```sh
+kubectl port-forward svc/elasticsearch-master 9200
+```
+
+The command waits until it is stop using <kbd>CTRL+C</kbd> kill signal, so we
+will need to create the `statements` index from another terminal using Curl:
+
+```
+curl -k -X PUT https://elastic:<PASTE PASSWORD HERE>@localhost:9200/statements
+```
+
+> You need to substitute the `<PASTE PASSWORD HERE>` placeholder with the
+> `elastic` user password we've fetched previously.
+
+> Note that our request target the `localhost` server name, since all traffic
+> on port 9200 is forwarded to the Elasticsearch service running in our
+> Kubernetes cluster.
+
+### Ralph
+
+**FIXME**: as long as Ralph Helm chart is not published in a public repository,
+one will need to clone Ralph's project first (and possibly checkout the
+`add-helm-chart` branch).
```

Comment:
```suggestion
As long as Ralph Helm chart is not published in a public repository,
one will need to clone Ralph's project first.
```

Reply:
Hello! I have created a pull request at [openfun/charts](https://github.com/openfun/charts) to use GitHub Pages as a Helm Repository host, this would resolve this. https://github.com/openfun/charts/pull/3

Using that as Helm Repository it would be required to move the Ralph Helm chart located in this repository to [openfun/charts](https://github.com/openfun/charts).

Reply:
Oh wow, thanks! We had in mind to publish our charts to Artifact Hub or alike, but given your proposal, we definitely need to re-think about it.

Reply:
I guess we chose to push our charts to DockerHub, right?